OpenRA.Game/IUtilityCommand.cs

```diff
@@ -20,6 +30,6 @@ public interface IUtilityCommand
 
 		bool ValidateArguments(string[] args);
 
-		void Run(ModData modData, string[] args);
+		void Run(Utility utility, string[] args);
```

Comment:
You might as well add the `[Traits.RequireExplicitImplementation]` attribute here when you already did all the work.